### DISCUSSION TITLE
LoRa Ra-01 + LoRa ES32  need help

### DISCUSSION THREAD

**Discussion by Bburn1** [sandeepmistry/arduino-LoRa#681] [Q&A]
Good evening. Need help to make friends ES32 and Ra-01. Namely, set the same AirRate ( SpreadingFactor / SignalBandwidth ), since they do not communicate with each other. There are no transmission problems between two es32 or ra-01 modules. And ra-01 and ES32 LoRa do not transmit data between themselves

**Comment by Bburn1** [comment]
LoRa E32*.   E32 and ra-01 have a 1278 chip

**Comment by dfs-brent** [comment]
I am trying to get this to work on a seeed xiao esp32c3.  I have commented out the version verification to get it to transmit.  It is looking for a version of 0x10 while my xiao has a version of 0x12.  Why are you requiring this version and will it work with newer versions? Thanks!

**Reply by dfs-brent** [reply]
Correction, you are looking for version == 0x12, my xiao has a version of 0x0A.  I guess mine is older.  Any chance it would work anyway?
